components/Comment/CommentFeed.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import Comment from "./Comment";
+import { Comment as CommentType } from "./lib/types";
+import CommentEditor from "~/components/Comment/CommentEditor";
+import {
+  createCommentAPI,
+  updateCommentAPI,
+  fetchCommentsAPI,
+} from "./lib/api";
+import { ID } from "~/config/types/root_types";
+import replaceComment from "./lib/replaceComment";
+import findComment from "./lib/findComment";
+
+type Args = {
+  unifiedDocumentId: ID;
+};
+
+const CommentFeed = ({ unifiedDocumentId }: Args) => {
+  const [comments, setComments] = useState<CommentType[]>([]);
+  const [isFetching, setIsFetching] = useState(true);
+
+  useEffect(() => {
+    const _fetchComments = () => {
+      setIsFetching(true);
+      fetchCommentsAPI({ unifiedDocumentId }).then(
```

Comment:
I wonder -- should we move away from .then syntax and instead 

```
const _fechComments = async () => {
     ...
     const comments = await fetchCommentsAPI(...);
     setComments(comments);
     setIsFetching(false);
}
```

Something like this, just a thought

Reply:
I actually prefer async/await syntax since it avoids [pyramid of doom](https://en.wikipedia.org/wiki/Pyramid_of_doom_(programming)). I updated in subsequent PR https://github.com/ResearchHub/researchhub-web/blob/aeec3fa455750c4b63ab57a166ba63f16aa2904d/components/Comment/CommentFeed.tsx#L37